tests/pytests/unit/modules/test_schedule.py

```diff
@@ -30,31 +32,85 @@ def sock_dir(tmp_path):
     return str(tmp_path / "test-socks")
 
 
+@pytest.fixture
+def schedule_config_file():
+    config_dir = schedule.__opts__.get("conf_dir", None)
```

Comment:
It looks like you are duplicating code here from the ` _get_schedule_config_file` function. Why not just call the ` _get_schedule_config_file()` function?